apps/auction/test/my_active_bids.coffee

```diff
@@ -0,0 +1,61 @@
+benv = require 'benv'
+_ = require 'underscore'
+Backbone = require 'backbone'
+template = require('jade').compileFile(require.resolve '../templates/my_active_bids.jade')
+bidderPositions = require './lot_standings'
+
+describe 'my active bids auction page template', ->
+  before (done) ->
+    @baseData = -> bidderPositions
+    benv.setup ->
+      benv.expose $: benv.require 'jquery'
```

Comment:
👌  this is the proper way to set up with benv. Good work! View tests like this is usually the hardest part of groking this stack.